tests/lib/util/path-util.js

```diff
@@ -74,7 +74,7 @@ describe("pathUtil", () => {
             const filePath = "file/path.js";
             const basePath = "/absolute/file";
 
-            sinon.stub(process, "cwd", () => "/absolute/");
+            sinon.stub(process, "cwd").callsFake(() => "/absolute/");
```

Comment:
It'd be more idiomatic to use `sinon.stub(process, "CWD").returns("/absolute/")` here.